assets/api/main.php

```diff
@@ -79,9 +78,9 @@ function saveimg() {
 		$res=checkuser();
 		$rights=intval($res[1]);
 		if ($rights==0) {echo '-18';exit;}
-		$con=mysql_connect("localhost","root","19951025");
+		dbconnect;
                 mysql_query("SET NAMES 'UTF8'");
```

Comment:
Identation. Or maybe we even don't need this if we add `mysql_query("SET NAMES 'UTF8'");` inside of `dbconnect`.